### PR TITLE
Change in detail view

### PR DESCRIPTION
**Pull request by BipinjotKaur**

### CODE REVIEW

**Review by Nauna — APPROVED** [review]

File: src/foam/nanos/crunch/ui/CapablePayloadROView.js [hunk 1]
```diff
@@ -115,7 +116,11 @@ foam.CLASS({
 
         // OPTION TWO
         // view.start().tag(this.VerticalDetailView, { data: data }).end()
-        view.start().add(data).end()
+        view.start()
```
Comment (kgrgreer) [inline src/foam/nanos/crunch/ui/CapablePayloadROView.js]: Do you need the start() and end()? Can you just do view.tag();  Also, don't forget the closing ; (I see that it was previously missing also.)

File: src/foam/nanos/crunch/ui/CapablePayloadROView.js [hunk 2]
```diff
@@ -115,7 +116,11 @@ foam.CLASS({
...
+          .tag(this.FlexSectionedDetailView, {
+            data: this.capableObj.capablePayloads[i].data
+          })
+        .end()
```
Comment (sarthak-marwaha) [inline src/foam/nanos/crunch/ui/CapablePayloadROView.js]:
```suggestion
        view
          .tag(this.FlexSectionedDetailView, {
            data: this.capableObj.capablePayloads[i].data
          })
```
dont need the wrapping .start().end() here as tag() adds it's own div

File: src/foam/nanos/crunch/ui/CapablePayloadROView.js [hunk 3]
```diff
@@ -116,11 +116,7 @@ foam.CLASS({
 
         // OPTION TWO
         // view.start().tag(this.VerticalDetailView, { data: data }).end()
-        view.start()
-          .tag(this.FlexSectionedDetailView, {
-            data: this.capableObj.capablePayloads[i].data
-          })
-        .end() 
+        view.tag(this.FlexSectionedDetailView, {data: this.capableObj.capablePayloads[i].data})
```
Comment (kgrgreer) [inline src/foam/nanos/crunch/ui/CapablePayloadROView.js:119]: Good. Still missing closing ;

Comment (Nauna) [reply src/foam/nanos/crunch/ui/CapablePayloadROView.js:119]: I'll add before requesting merge to development

**Pull request merged**